### PR TITLE
ADR guidance

### PR DESCRIPTION
**Pull request by galund**
Update guidance for making and documenting architectural decisions
- describe a working ADR process/lifecycle that uses GitHub
- note that tracking implementation is important
- mention that other architecture document styles are available and important too

### CODE REVIEW

**Review by jamesmelville-gds — DISMISSED** [review]

File: source/standards/architecture-decisions.html.md.erb [hunk 1]
```diff
@@ -19,24 +19,100 @@ make. For example, understanding whether a particular choice was made for the
...
+
+Amazon Web Services (AWS) also provide
+[useful advice](https://docs.aws.amazon.com/prescriptive-guidance/latest/architectural-decision-records/welcome.html)
+about how and why the ADR process should be adopted.
+
+## Lifecycle of an ADR
+
+In a new project or one with a lot of decisions to be made, consider setting up
+regular ADR discussion and review meetings. This will give the team time to work
+though these decisions.
```
Comment (paroxp) [inline source/standards/architecture-decisions.html.md.erb:48]:
```suggestion
through these decisions.
```

File: source/standards/architecture-decisions.html.md.erb [hunk 2]
```diff
@@ -19,24 +19,100 @@ make. For example, understanding whether a particular choice was made for the
...
+
+### "Proposed" status
+
+Assuming you're using GitHub for version control, there is no need to record a 'status' for
+an ADR when it is first proposed.
+The status of your ADR's pull request (PR) reflects the status of the decision until it has been
+accepted.
+
+Consider using GitHub's "draft PR" feature especially if you have lots of decisions at varying
+levels of readyness.
```
Comment (paroxp) [inline source/standards/architecture-decisions.html.md.erb:60]:
```suggestion
levels of readiness.
```

**Comment by huwd** [comment]
@paroxp, @galund, @jamesmelville-gds looks like these conversations have all been resolved.
Anyone happy to ✅ or suggest new changes?

**Review by jamesmelville-gds — APPROVED** [review]

**Comment by galund** [comment]
@louzoid-gds says "LGTM" 🥳

**Pull request merged**